### PR TITLE
Overlays

### PR DESCRIPTION
**Pull request by Incompleteusern**
- [ ] `HealthBarOverlay`
- [ ] `BubbleOverlay`
- [ ] `ExpBarOverlay`

### CODE REVIEW

File: common/src/main/java/com/wynntils/core/features/FeatureRegistry.java [hunk 1]
```diff
@@ -15,37 +15,35 @@
...
+import java.util.EnumMap;
 import java.util.LinkedList;
 import java.util.List;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
 /** Loads {@link Feature}s */
 public class FeatureRegistry {
-    private static final List<Feature> FEATURES = new LinkedList<>();
-    private static final List<Overlay> OVERLAYS = new LinkedList<>();
+    private static final List<Feature> features = new LinkedList<>();
```
Comment (magicus) [inline common/src/main/java/com/wynntils/core/features/FeatureRegistry.java]: static final members are conventionally named in upper case. Why change this?

Comment (Incompleteusern) [reply common/src/main/java/com/wynntils/core/features/FeatureRegistry.java]: I've always found lower case more preferable but I am fine with upper case. I will revert that change

**Comment by Incompleteusern** [comment]
These overlays are proving tough to tackle, going to try and get only `GammaOverlay` done

**Comment by magicus** [comment]
So what does GammaOverlay do? Add a text to tell user gamma override is enabled? Did we have something like that in legacy Wynntils? 

(I can't understand why that'd be useful, but as long as it's possible to turn off -- and off by default! -- I'm ok with it).

**Comment by Incompleteusern** [comment]
yeah this is from legacy wynntils, and will basically be a proof of concept till configs appear

**Pull request closed without merging**